Service/RequestService.php

```diff
@@ -497,6 +497,22 @@ public function requestHandler(array $data, array $configuration): Response
                 } else {
                     //$this->data['query']['_schema'] = $this->data['endpoint']->getEntities()->first()->getReference();
                     $result = $this->cacheService->searchObjects(null, $filters, $allowedSchemas['id']);
+
+                    if (count($allowedSchemas['id']) === 1) {
+                        $entity = $this->entityManager->getRepository('App:Entity')->find($allowedSchemas['id'][0]);
+
+                        if (empty($this->data['endpoint']->getThrows()) === false) {
```

Comment:
Just in case, you might want to check if isset($this->data['endpoint']) === true